presto-docs/src/main/sphinx/installation/spark.rst

```diff
@@ -0,0 +1,69 @@
+=========================
+Executing Presto on Spark
+=========================
+
+When there is a need to execute long running queries, Presto offers
+an option to scale the resource management using Spark, as it doesn'f offer
```

Comment:
nit: 
- scale the resource management using Spark -> scale the resource management *by* using Spark
- as it doesn'f offer -> as it doesn't offer

Reply:
changed , thanks